data/map planets.txt

```diff
@@ -593,6 +593,13 @@ planet "Carbuncle Station"
 		threshold 5500
 		fleet "Large Republic" 10
 
+planet Cassandra
+	attributes paradise uninhabited
+	landscape land/mars4
+	description `In the late 2300s, Cassandra was the subject of one of the most ambitious terraforming projects performed by humanity up to that point; an attempt to rapidly modify its biosphere and jumpstart the process of making the planet more than perfect for human habitation.`
+	description `The project never came to be. For an as-yet-unexplained reason, some collapse or chain of destruction occurred during the terraforming project that made Cassandra uninhabitable, and ever since Cassandra has been discreetly left out of the public record by the terraforming industry.`
```

Comment:
```suggestion
	description `	The project never came to be. For an as-yet-unexplained reason, some collapse or chain of destruction occurred during the terraforming project that made Cassandra uninhabitable, and ever since Cassandra has been discreetly left out of the public record by the terraforming industry.`
```